src/main/java/io/github/enkarin/bookcrossing/chat/service/CorrespondenceService.java

```diff
@@ -2,16 +2,12 @@
 
 import io.github.enkarin.bookcrossing.chat.dto.MessageDto;
 import io.github.enkarin.bookcrossing.chat.dto.UsersCorrKeyDto;
-import io.github.enkarin.bookcrossing.chat.dto.ZonedUserCorrKeyDto;
 import io.github.enkarin.bookcrossing.chat.model.Correspondence;
 import io.github.enkarin.bookcrossing.chat.model.Message;
 import io.github.enkarin.bookcrossing.chat.model.UsersCorrKey;
 import io.github.enkarin.bookcrossing.chat.repository.CorrespondenceRepository;
 import io.github.enkarin.bookcrossing.chat.repository.MessageRepository;
-import io.github.enkarin.bookcrossing.exception.CannotBeCreatedCorrespondenceException;
-import io.github.enkarin.bookcrossing.exception.ChatAlreadyCreatedException;
-import io.github.enkarin.bookcrossing.exception.ChatNotFoundException;
-import io.github.enkarin.bookcrossing.exception.UserNotFoundException;
+import io.github.enkarin.bookcrossing.exception.*;
```

Comment:
It's better to avoid star import

Reply:
it would be necessary to disable the optimization of imports when committing